mod-inventory/src/main/resources/domain/mod-inventory/inventory-junit.feature

```diff
@@ -4,16 +4,33 @@ Feature: mod-inventory integration tests
     * url baseUrl
     * table modules
       | name                                |
+      | 'okapi'                             |
       | 'mod-login'                         |
       | 'mod-permissions'                   |
-      | 'mod-inventory'                      |
+      | 'mod-inventory'                     |
       | 'mod-inventory-storage'             |
 
+
     * table adminAdditionalPermissions
       | name                                |
 
     * table userPermissions
-      | name                                          |
+      | name                                                             |
+      |'inventory.instances.item.get'                                    |
+      |'inventory.items.collection.get'                                   |
+      |'inventory.instances.collection.get'                              |
+      |'inventory-storage.instances.item.post'                           |
+      |'inventory.instances.item.put'                                    |
+      |'inventory-storage.instances.item.get'                            |
+      |'inventory-storage.preceding-succeeding-titles.collection.get'    |
+      |'inventory-storage.preceding-succeeding-titles.item.post'         |
+      |'inventory-storage.preceding-succeeding-titles.item.put'          |
+      |'inventory-storage.preceding-succeeding-titles.item.delete'       |
+      |'inventory-storage.instance-relationships.collection.get'         |
+      |'inventory-storage.instance-relationships.item.post'              |
+      |'inventory-storage.instance-relationships.item.put'               |
+      |'inventory-storage.instance-relationships.item.delete'            |
+      |'inventory.instances.item.post'                                   |
```

Comment:
Why does this test need this many permissions when it is created an instance via the API, which should only require  `inventory.instances.item.post`?